examples/custom/font-awesome/named_icon.py

```diff
@@ -67,17 +65,4 @@
     "vine", "vk", "volume-down", "volume-off", "volume-up", "warning", "wechat", "weibo", "weixin", "wheelchair", "wifi",
     "windows", "won", "wordpress", "wrench", "xing", "xing-square", "yahoo", "yelp", "yen", "youtube", "youtube-play",
     "youtube-square",
-]
-
-__doc__ += '''
-.. raw:: html
-
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
-
-    <ul class="fa-ul" style="list-style: none !important;">
-'''
-
-for icon_name in __icons__:
-    __doc__ += '        <li><i class="fa-li fa fa-%s" style="list-style-type: none !important;"></i>%s</li>\n' % (icon_name, icon_name)
-
-__doc__ += '    </ul>'
+])
```

Comment:
Can we leave a `bokeh/icons.py` for some time that just has a `warnings.warn` that documents when the module was removed, and points people at the example?